docs/contribute.md

```diff
@@ -6,17 +6,17 @@ redirect_from: "/Contribute"
 
 You can contribute in several ways to AzerothCore:
 
-- [commenting an open issue](Contribute#how-to-comment-an-issue)
+- [commenting an open issue](contribute#how-to-comment-an-issue)
 
-- [opening an issue](Contribute#how-to-open-an-issue)
+- [opening an issue](contribute#how-to-open-an-issue)
 
 - [testing a pull request](How-to-test-a-PR)
 
 - [testing DB-only changes](How-to-test-DB-only-changes)
 
-- [creating a pull request](Contribute#how-to-create-a-pull-request)
+- [creating a pull request](contribute#how-to-create-a-pull-request)
 
-- [improving our wiki](Contribute#improve-the-wiki)
+- [improving our wiki](contribute#improve-the-wiki)
```

Comment:
fine for me but don't forget to add redirects to the old pages

Reply:
@FrancescoBorzi,
As far as I know, the only way to make a redirect like so with the limitation of Markdown here is to rename the file. The link was broken and lead to nowhere.

Do you suggest a better idea on how to accomplish this without renaming the file?

Reply:
@zanthed 

you can use `redirect_from:` at the top of the page, for example:

https://raw.githubusercontent.com/azerothcore/wiki/master/docs/installation.md

Reply:
@FrancescoBorzi,
I figured that I made a better solution of instead of going to the link, just go to the header.

See if this is fine for you: https://github.com/Zanthed/wiki/commit/f9729336cd6ab70e564d326593726398d5f127e4#diff-676bbff8303d858b713b60da214eb6691d9cd99be60b55f3d30fa37b01b82cf1L9

It works the same way here without messing with redirects.

Reply:
you still need the redirect_from, because other links outside the wiki can still point to the old page

Reply:
that file already has redirect_from. All files that was once in an uppercase naming has it. So I don't see an issue with these changes